c2corg_api/views/__init__.py

```diff
@@ -1,6 +1,32 @@
 import collections
 import datetime
 from colander import null
+from pyramid.httpexceptions import HTTPError, HTTPNotFound
+from pyramid.view import view_config
+from cornice import Errors
+from cornice.util import json_error
+
+
+@view_config(context=HTTPNotFound)
+@view_config(context=HTTPError)
```

Comment:
By Pyramid in this case. When you raise/return HTTPConflict, Pyramid sets the title (see [here](http://docs.pylonsproject.org/projects/pyramid/en/latest/_modules/pyramid/httpexceptions.html#HTTPConflict)).